.github/workflows/flaky-tests-stalebot-grooming.yml

```diff
@@ -0,0 +1,23 @@
+name: Flaky test auto-grooming
+# Automatically closes flaky test issues when they are over a week old with no activity.
+on:
+    schedule:
+        - cron: '0 0 * * *'
+
+jobs:
+    stale:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Close Stale Issues
+              uses: actions/stale@v9.0.0
+              with:
+                  only-labels: 'metric: flaky e2e test, team: Vortex'
```

Comment:
The `metric` label only gets assigned to these issues, so that should filter it enough that only these issues are evaluated.